src/Hubbup.IssueMoverClient/Pages/IssueMover.cshtml

```diff
@@ -27,7 +26,7 @@
             </div>
             <div class="col-md-1">
                 <div class="input-group">
-                    <input type="button" class="btn btn-info @(State.ShouldMoveButtonBeEnabled() ? "" : "disabled")" disabled="@(!State.ShouldMoveButtonBeEnabled())" value="Go" onclick="@State.OnMoveButtonClick" />
+                    <input type="button" class="btn btn-info" disabled="@(!State.ShouldMoveButtonBeEnabled())" value="Go" onclick="@State.OnMoveButtonClick" />
```

Comment:
For Bootstrap don't I need the `disabled` CSS class also applied?

Reply:
Oh, maybe you do. If so please ignore and close this.

I tried it and as far as I remember it looked correctly disabled without repeating that info in the CSS class, but maybe there was something more subtle in the styling that I missed.